Sources/Widgets/Widgets3D/ResliceCursorWidget/example/index.js

```diff
@@ -214,14 +215,21 @@ reader.setUrl(`${__BASE_PATH__}/data/volume/LIDC2.vti`).then(() => {
                 .getWidgetState()
                 .getActiveViewName();
               const currentViewName = getViewPlaneNameFromViewType(viewType);
+              const currentUpdateMethodName = widget
+                .getWidgetState()
+                .getUpdateMethodName();
+              const keepFocalPointPosition =
+                activeViewName !== currentViewName &&
+                canUpdateFocalPoint &&
```

Comment:
shouldn't `canUpdateFocalPoint` be already set to false when `currentUpdateMethodName !== InteractionMethodsName.RotateLine` ?

Reply:
Good point. After rechecking comments, the behavior class wasn't correct. 
With this correction, we have now:
- translation (axe, center): move reslice cursor center
- rotation: don't move reslice cursor center

Reply:
I like that